frontend/src/containers/projects.tsx

```diff
@@ -128,7 +128,7 @@ export const ProjectPage = (): JSX.Element => {
                   )}
 
                   <FormGroup className="text-center pt-4">
-                    <Button color="primary" onClick={onFilterSubmit}>
+                    <Button color="primary" type="submit" onClick={onFilterSubmit}>
```

Comment:
Same change here

Reply:
@gtindo i made the change. should i in this case make a new PR or is it okay?

Reply:
It's OK